packages/teleterm/src/ui/hooks/useLoggedInUser.ts

```diff
@@ -0,0 +1,23 @@
+import { useAppContext } from 'teleterm/ui/appContextProvider';
+import { LoggedInUser } from 'teleterm/services/tshd/types';
+
+export function useLoggedInUser() {
+  const ctx = useAppContext();
+
+  ctx.clustersService.useState();
+  ctx.workspacesService.useState();
+
+  function getLoggedInUser(): LoggedInUser | undefined {
```

Comment:
What's the reason behind wrapping this in a function?

Reply:
IIRC it was just refactored from the previous iteration of where we were getting `loggedInUser` in a different hook. I'll take it out of a function